python/src/uagents/context.py

```diff
@@ -634,6 +686,22 @@ async def send_raw_exchange_envelope(
             endpoint="",
         )
 
+    @staticmethod
+    async def dispatch_sync_response_envelope(env: Envelope) -> MsgStatus:
+        await dispatcher.dispatch(
+            env.sender,
+            env.target,
+            env.schema_digest,
+            env.decode_payload(),
+            env.session,
+        )
+        return MsgStatus(
```

Comment:
This means that the `MsgStatus` for the sent message is delivered/received after the actual response has already been handled?

I guess it's not of any relevance, just trying to follow this sync async flow.

Reply:
Right, so the message is going out to the destination as a sync response and the status is going back to the sender as a successful sync delivery.